comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -108,13 +119,27 @@ func newRemoteConfigClient(deps dependencies) (rcclient.Component, error) {
 		clientHA:      clientHA,
 	}
 
+	deps.Lc.Append(fx.Hook{
+		OnStart: func(context.Context) error {
+			if config.IsRemoteConfigEnabled(config.Datadog) {
+				return rc.start()
```

Comment:
We can no longer use return here since the `start` function does not return any value